src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -16,6 +16,7 @@
 -->
 
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <application>sp19-codeu-9-3673</application>
```

Comment:
It should not be necessary to set this tag; the `pom.xml` file specifies how to obtain the application name automatically.